### PR TITLE
Adds bootstrap grid and refactores CSS #266

### PR DESCRIPTION
**Pull request by roosdebildt**
Implementation of the bootstrap grid for the big components: header, content and footer. Didn't implement rows and columns for all the specific views yet, but this is the base for all improvements.

*update*
I think I'm done! @alicetragedy could you review?

### CODE REVIEW

**Comment by alicetragedy** [comment]
@roosdebildt just one thing that came to mind: could you update the title of the PR to something like [WIP] or [do not merge yet] at the beginning, so it's clear when looking at the list of pull requests which ones don't need reviewing yet? would be great :+1:

**Comment by roosdebildt** [comment]
done! thanks for the tip! :+1:

**Comment by alicetragedy** [comment]
hey @roosdebildt, I've just looked through the code and checked it out locally and it looks SO. MUCH. BETTER! :sparkles: 
Just a quick question: you'll then go into minor improvements if possible next, right? I just noticed minor things (like the pagination which, on smaller screens, we can then change to something else like endless scrolling) and the tables (which we need to think about how to portray best on smaller screens) and that team dropdown which is still looking a little funky. I understand that's not a huge concern right now, and I am super glad to have this look so much better on mobile!
I'm very happy to merge as soon as possible – are you okay with that @carpodaster?

**Comment by F3PiX** [comment]
:+1:  voor Roos!!

**Comment by roosdebildt** [comment]
@alicetragedy when it comes to the incremental improvements such as tables, the community page etc: I will do that if I have time for it. Right now it's about the basis being there so that things can be improved gradually. But first... the dashboard :)

**Comment by roosdebildt** [comment]
p.s. I think @carpodaster doesn't have Internet (on his computer) atm, so if you're OK with merging... could we do that?

File: app/views/layouts/_header.html.slim [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
-    = link_to ' ', root_path
+section#header  
+  header.container
+    div.row
+      div.col-xs-7
+        = link_to image_tag("logo_subnavi.png"), root_path
+      div.col-xs-5
+        div.text-right
+          a href="#" class="button-burger" id="nav-toggle" data-toggle="collapse" data-target="#menu"
+              = icon('reorder')
```
Comment (marcgreenstock) [inline app/views/layouts/_header.html.slim]: too much indentation, i guess it works, but slim could complain about it

File: app/assets/stylesheets/global.css.sass [hunk 2]
```diff
@@ -2,169 +2,130 @@
...
-  nav
-    clear: right
-    float: right
-    @include clearfix
+  nav.user
+    color: $darkgray
 
-    &.top
+  nav.top
+    @media screen and (min-width: 992px)
```
Comment (marcgreenstock) [inline app/assets/stylesheets/global.css.sass]: does it work if you use the `$screen-md` variable in less?

e.g.
```sass
@media screen and (min-with: $screen-md)
```

it's a variable in [bootstrap-sass](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L307) and defaulted at `992px`

File: app/assets/stylesheets/global.css.sass [hunk 3]
```diff
@@ -2,169 +2,130 @@
...
+        &:last-child:after, &:first-child:after
+          content: none
+      a
+        color: white
+        text-decoration: none
 
-  &.page
-    font-size: 0.9em
+    // Changes bootstraps default collapsing breakpoint
+    @media (min-width: 768px) and (max-width: 991px)
```
Comment (marcgreenstock) [inline app/assets/stylesheets/global.css.sass]: same as my comment above you can use [$screen-sm](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L300) and [$screen-sm-max](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L321)

File: app/assets/stylesheets/global.css.sass [hunk 4]
```diff
@@ -2,169 +2,130 @@
...
+    width: 35px
+    height: 35px
+    font-size: 24px
+    text-decoration: none
+    color: $railsred
+    background: $white
+    border-radius: 50%
+    overflow: hidden
+    margin-top: 12px
+    @media screen and (min-width: 992px)
```
Comment (marcgreenstock) [inline app/assets/stylesheets/global.css.sass]: Again, here you can use `$screen-md`

**Comment by marcgreenstock** [comment]
Just a few comments, mostly on levering builtin bootstrap variables, otherwise exceptional work here! Thank's so much!

**Comment by alicetragedy** [comment]
@roosdebildt if you could take a look at @marcgreenstock's suggestions and make possible changes accordingly, I can definitely merge after that. Having been gone for the last week I didn't realise Carsten was without computer so thanks for letting me know :)
Feel free to ping me when the changes have been made! :tada:

File: app/assets/stylesheets/global.css.sass (hunk 2 above)
Comment (roosdebildt) [inline app/assets/stylesheets/global.css.sass]: ahhh @marcgreenstock thank you so much! I was unaware that you had to change the `@` for `$` to make it work in sass. Therefore, I just put the screen sizes in manually. But this is way better of course and good to know! :+1:

File: app/views/layouts/_header.html.slim (hunk 1 above)
Comment (roosdebildt) [inline app/views/layouts/_header.html.slim]: done!

**Comment by roosdebildt** [comment]
thanks @marcgreenstock for the tips! super helpful. I adjusted all your comments, should be all good now. I do see there's a merge conflict now, it shouldn't be too difficult but I've never done that :flushed: How do we do this now?

**Pull request merged**